### PR TITLE
fix:Solve the left and right key bug. 

### PR DESCRIPTION
**Pull request by q200892907**
Solve the left and right key bug. When adding an input box at the head or tail of the editor, it cannot be moved.

### CODE REVIEW

**Review by Xazin — APPROVED** [review]
CC @LucasXu0

File: lib/src/editor/editor_component/service/keyboard_service_widget.dart [hunk 1]
```diff
@@ -52,8 +51,7 @@ class KeyboardServiceWidgetState extends State<KeyboardServiceWidget>
         return true;
       },
     );
-    editorState.service.selectionService
-        .registerGestureInterceptor(interceptor);
+    editorState.service.selectionService.registerGestureInterceptor(interceptor);
```
Comment (Xazin) [inline lib/src/editor/editor_component/service/keyboard_service_widget.dart:54]: Seems like your formatting options are different than default 80 character line length.

**Review by Xazin — CHANGES_REQUESTED** [review]

**Comment by LucasXu0** [comment]
@q200892907 The format step failed. PTAL.

**Comment by q200892907** [comment]
> @q200892907 The format step failed. PTAL.

Restored default format

**Pull request merged**